### ISSUE TITLE
Linux installation problem with VirtualEnv

### ISSUE THREAD

**Issue by spiderhacker** [tensorflow/tensorflow#133]
I'm using virtual-env to install tensorflow. I've successfully created a virtual-env and when trying to install tensorflow using pip, i'm getting this error.
Any suggestions ?? Thanks.

```
Unpacking /home/ayan/Desktop/tensorflow-0.5.0-cp27-none-linux_x86_64.whl
  Running setup.py egg_info for package from file:///home/ayan/Desktop/tensorflow-0.5.0-cp27-none-linux_x86_64.whl
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
    IOError: [Errno 2] No such file or directory: '/tmp/pip-ULRUTP-build/setup.py'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
  File "<string>", line 16, in <module>
IOError: [Errno 2] No such file or directory: '/tmp/pip-ULRUTP-build/setup.py'
----------------------------------------
Command python setup.py egg_info failed with error code 1 in /tmp/pip-ULRUTP-build
Storing complete log in /home/ayan/.pip/pip.log
```

**Comment by mrob27** [comment]
Also happens on Mac: https://github.com/tensorflow/tensorflow/issues/46
 Also reported on stackexchange: http://stackoverflow.com/questions/33623453/pip-installation-error-no-such-file-or-directory-setup-py

They claim that "with python 2.7.10, python-dev and pip 1.5.6, tensorflow will be successfully installed using the pip method."

What version of pip are you using (type "pip --version")

**Comment by spiderhacker** [comment]
My pip is up-to-date ("pip --version" gives 7.1.2). But my python version is 2.7.4. Is that the problem ?

**Comment by mrob27** [comment]
That's kind of odd, since I got it to work in Python 2.7.3. (And it fixed [my version of the bug](https://github.com/tensorflow/tensorflow/issues/46) )

Were you still within the virtualenv? To be clear, the full command sequence would be:

```
# assuming we want to to be running virtualenv within ~/tensorflow
virtualenv --system-site-packages ~/tensorflow
cd ~/tensorflow
source bin/activate
# we are in the virtual environment, and ./bin is at the beginning of $PATH
pip install --upgrade pip
pip --version
# In the following use "/cpu/" or "/gpu" as desired
pip install --upgrade https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.5.0-cp27-none-linux_x86_64.whl
# (we would continue with tensorflow setup/testing here)
# when done, we could exit the shell completely, or just leave virtualenv with this command:
deactivate
# Now the prompt and $PATH are back to normal
```

**Comment by spiderhacker** [comment]
Its still the same after upgrading everything. Python is now 2.7.10, pip is 7.1.2.
VirtualEnv is still activated in shell. And the sequence is absolutely same as you showed.
After downloading the .whl file, I'm getting the same error.

**Comment by mrob27** [comment]
Darn, I'm out of ideas. But I will edit my comment above to remove the suggestion that "this worked for me so it should work for you".

**Comment by vincentvanhoucke** [comment]
Did this issue get resolved? A lot of things have changed with 0.6.0.
